docs/content/advanced/development.md

```diff
@@ -17,6 +17,53 @@ This section will collect how-to, notes and development documentation
 
 We use conventional commits and semantic versioning. Please follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) specification when writing commit messages.
 
+## LocalAI Project Structure
+
+**LocalAI is made of multiple components, developed in multiple repositories:**
+
+The core repository, containing the primary `local-ai` server code, gRPC stubs, this documentation website, and docker container building resources are all located at [mudler/LocalAI](https://github.com/mudler/LocalAI).
+
+As LocalAI is designed to make use of multiple, independent model galleries, those are maintained seperately. The following public model galleries are available for use:
+
+* [go-skynet/model-gallery](https://github.com/go-skynet/model-gallery) - The original gallery, the `golang` huggingface scraper ran into limits and was largely retired, so this now holds handmade yaml configs
+* [dave-gray101/model-gallery](https://github.com/dave-gray101/model-gallery) - An automated gallery designed to track HuggingFace uploads and produce best-effort automatically generated configurations for LocalAI. It is designed to produce one LocalAI gallery per repository on HuggingFace.
```

Comment:
that's a great addition too, I don't want that to go down unnoticed, maybe also add it in the README? https://github.com/mudler/LocalAI#-community-and-integrations